src/components/AuthButton.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../views/Button/index";
+import { Logout } from "../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
```

Comment:
I can still see a bind statement, a constructor and ES5 functions. @mikenthiwa @Bileonaire